src/profile-logic/mozilla-symbolication-api.js

```diff
@@ -220,8 +227,9 @@ function getV5ResultForLibRequest(
 // not have symbol information for a given library, the LibSymbolicationResponse
 // for that library will have .type === 'ERROR'.
 export async function requestSymbols(
+  symbolSupplierName: string,
   requests: LibSymbolicationRequest[],
-  symbolsUrl: string
+  callback: (path: string, requestJson: string) => Promise<MixedObject>
```

Comment:
I believe the name `callback` isn't descriptive enough, what do you think of renaming it `symbolFetcherCallback` or something similar?
Also please describe it a bit in the comment above.

Reply:
Good suggestions, will do.